### PR TITLE
Make fokia accept user provided arguments in case of direct usage.

### PR DESCRIPTION
**Pull request by pkoro**
# Description

The proposed modification allows the Fokia CLI user to define in a custom way (via command line arguments) the specifications of the to-be-provisioned lambda instance. 
# Implementation

Added import of `argparse` library within main section and allow the definition of command line arguments which can be used in order to define specifications for the provisioned VMs. An additional help section within each argument helps when the user calls the wrapper using the `-h` or the `--help` arguments. E.g.:

```
$ python lambda_instance_manager.py --help
usage: lambda_instance_manager.py [-h] [--master-name MASTER_NAME]
                                  [--slaves SLAVES]
                                  [--vcpus_master VCPUS_MASTER]
                                  [--vcpus_slave VCPUS_SLAVE]
                                  [--ram_master RAM_MASTER]
                                  [--ram_slave RAM_SLAVE]
                                  [--disk_master DISK_MASTER]
                                  [--disk_slave DISK_SLAVE]
                                  [--project-name PROJECT_NAME]

Okeanos VM provisioning

optional arguments:
  -h, --help            show this help message and exit
  --master-name MASTER_NAME
                        Name of Flink master VM [default: lambda-master]
  --slaves SLAVES       Number of Flink slaves [default: 1]
  --vcpus_master VCPUS_MASTER
                        Number of CPUs on Flink master [default: 4]
  --vcpus_slave VCPUS_SLAVE
                        Number of CPUs on Flink slave(s) [default: 4]
  --ram_master RAM_MASTER
                        Size of RAM on Flink master (in MB) [default: 4096MB]
  --ram_slave RAM_SLAVE
                        Size of RAM on Flink slave(s) (in MB) [default:
                        4096MB]
  --disk_master DISK_MASTER
                        Size of disk on Flink master (in GB) [default: 40GB]
  --disk_slave DISK_SLAVE
                        Size of disk on Flink slave(s) (in GB) [default: 40GB]
  --project-name PROJECT_NAME
                        ~okeanos Project [default: lambda.grnet.gr]
```

please review /cc @gouzouni625 @ioantsaf

### CODE REVIEW

**Comment by themiszamani** [comment]
please add system project as the default project. The lambda.grnet.gr is only for internal use.

**Comment by pkoro** [comment]
Default project is not `lambda.grnet.gr` anymore.

File: core/fokia/lambda_instance_manager.py [hunk 1]
```diff
@@ -179,7 +184,16 @@ def lambda_instance_destroy(instance_uuid, auth_token,
...
-    ansible_manager, provisioner_response = create_cluster(cluster_id=uuid.uuid4())
+
+    ansible_manager, provisioner_response = create_cluster(cluster_id=uuid.uuid4(),
+                                                           master_name=args.master_name,
+                                                           slaves=args.slaves,
+                                                           vcpus_master=args.vcpus_master,
+                                                           vcpus_slave=args.vcpus_slave,
+                                                           ram_master=args.ram_master,
+                                                           ram_slave=args.ram_slave,
+                                                           disk_master=args.disk_master,
```
Comment (stevelaskaridis) [inline core/fokia/lambda_instance_manager.py:195]: `disk_slave=args.disk_slave,` to be added.

Comment (pkoro) [inline core/fokia/lambda_instance_manager.py:195]: Good catch @stevelaskaridis . Thanks!

**Comment by stevelaskaridis** [comment]
+1

**Pull request merged**